governance/votes/Executive Vote - February 4, 2022.md

```diff
@@ -0,0 +1,116 @@
+* **UNIV2DAIETH-A** [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased from 1% from **2%** to **1%**.
+* **UNIV2WBTCETH-A** [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 1% from **3%** to **2%**.
+* **UNIV2USDCETH-A** [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 1% from **2.5%** to **1.5%**.
+* **GUNIV3DAIUSDC2-A** [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.25% from **0.5%** to **0.25%**.
+* **TUSD-A** [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 1% from **1%** to **0%**.
+
+**DIRECT-AAVEV2-DAI (Aave D3M) Target Borrow Rate Decrease**
+
+* **DIRECT-AAVEV2-DAI (Aave D3M)** [Target Borrow Rate (`bar`)](https://github.com/makerdao/dss-direct-deposit#configuration) will be decreased by 0.25% from **3.75%** to **3.5%**.
+
+**Maximum Debt Ceiling Changes**
+
+* **GUNIV3DAIUSDC1-A** [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be decreased by 400 million DAI from **500 million DAI** to **100 million DAI**.
+* **GUNIV3DAIUSDC2-A** [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 250 million DAI from **500 million DAI** to **750 million DAI**.
+
+**GUNIV3DAIUSDC2-A Target Available Debt Increase**
+
+* **GUNIV3DAIUSDC2-A** [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) will be increased by 40 million DAI from **10 million DAI** to **50 million DAI**.
+
+### Correction of Core Unit Budget Streams
+
+The previous Executive Vote contained errors relating to the DAI budget Streams for the StarkNet Engineering and Strategic Finance Core Units. The error has been detailed by the Protocol Engineering team on the MakerDAO forum [here](https://forum.makerdao.com/t/correction-to-last-weeks-executive/13022). In order to correct this error, the following changes will occur if this executive proposal passes.
+
+* The currently configured DAI budget streams for the StarkNet Engineering and Strategic Finance Core Units will be cancelled.
+* In order to retain access to the first month’s funds during the stream update, **we are transferring 42,917 DAI minus the amount claimable from the old contract**, at the time that this Executive Vote is executed to the [StarkNet Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP19) at 0x6D348f18c88D45243705D4fdEeB6538c6a9191F1. The maximum transfer amount of 42,917 DAI is equivalent to one month of funding under their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP47).
```

Comment:
"**we are transferring [...] old contract**, at the time". That solitary comma may be wrong there, I think. Several other possibilities:

- "42,917 (minus [...] at the time that this Executive [...]) to the StarkNet..."
- "--at the time that this Executive Vote is executed--"
- "(at the time that this Executive Vote is executed)"

Reply:
(Same with all the commas in the same structure that's repeated further down)

Reply:
Yeah, I wasn't fully happy with it either, I think the parentheses that you suggested might be better

Reply:
Learned I was slightly wrong on how this would be accomplished, so that part of the sentence wasn't needed 😅 thanks for working it out tho!